src/mmw/js/src/modeling/gwlfe/runoff/views.js

```diff
@@ -3,11 +3,11 @@
 var $ = require('jquery'),
     _ = require('lodash'),
     Marionette = require('../../../../shim/backbone.marionette'),
+    App = require('../../../app.js'),
```

Comment:
This is a poor design choice that we've never been able to get away from. Since you often need a reference to the current project, or the current AoI, we put that in the top level `App`, but then you often end up importing it all over the project, which can create circular dependencies if you're not careful. Thankfully, that's not the case for this file.